reference/interbit-cli/manifest/app.adoc

```diff
@@ -0,0 +1,27 @@
+= The `app` resolution object
+
+The `app` resolution object appears within an link:README.adoc[Interbit
+manifest file]. It describes the resolved configuration for an Interbit
+application, describing which blockchain should load the application,
+the filesystem path to the built application, and the blockchains that
+the browser nodes should load.
+
+[source,js]
+----
+{% include "app.js" %}
+----
+
+
+== Properties
+
+[horizontal]
+[.app.p]`appChain` [.api.t]__String__::
+The alias of the blockchain that should load the application.
+
+[.app.p]`buildLocation` [.api.t]__String__::
+The filesystem path to the directory containing the built application.
+The application, when run, creates a browser node that connects to this
```

Comment:
We can't really say for certain what the app does here as it's decoupled from this manifest aside from writing this context in the start/deploy steps. We only know that we have given enough context for the app to connect to some chains from our manifest. What it does with that information is up to implementation of the app.

Reply:
Can I say that we use this information to update the application's `package.json` file?

Reply:
We don't update the `package.json`, we update the `index.html` file with context so the browser node can find peers and connect to chains.

Reply:
Interestingly, we also don't use this part of the manifest (yet). The writing to `index.html` occurs during the build step. In the future we plan to bundle the app and load it from a chain which is why this is here in the manifest.